daemons/Makefile

```diff
@@ -59,5 +59,6 @@ import-test:
             ./package_daemon.sh $$daemon; \
             python $$daemon/app.py; \
         done
+	python dss-gs-event-relay/main.py
```

Comment:
What happens here?

Reply:
This is a lint test. It's just a sanity check to see if the code in the module compiles and imports without triggering any errors.